src/runtime/entitlements-manager-test.js

```diff
@@ -1685,6 +1749,7 @@ describes.realWin('EntitlementsManager', {}, (env) => {
 
     it('should ignore malformed pushed entitlements', () => {
       storageMock.expects('set').withArgs('ents').never();
+      sandbox.stub(win, 'setTimeout').callsFake(() => {});
```

Comment:
`setTimeout` should return a `Number`. Just curious though, is this line still needed?
```suggestion
      sandbox.stub(win, 'setTimeout').returns(1);
```

Reply:
Changed to returns(1). Without a stub, setTimeout will fail with a `TypeError: Illegal invocation`, I think because it hits L449 on entitlements-manager.js and doesn't know what to do without an implementation of setTimeout?